analysisservice/src/test/java/nu/yona/server/analysis/service/AppActivitiesDtoTest.java

```diff
@@ -0,0 +1,34 @@
+/*******************************************************************************
+ * Copyright (c) 2018 Stichting Yona Foundation This Source Code Form is subject to the terms of the Mozilla Public License, v.
+ * 2.0. If a copy of the MPL was not distributed with this file, You can obtain one at https://mozilla.org/MPL/2.0/.
+ *******************************************************************************/
+package nu.yona.server.analysis.service;
+
+import static org.hamcrest.CoreMatchers.equalTo;
+import static org.junit.Assert.assertThat;
+
+import java.time.ZonedDateTime;
+import java.util.Arrays;
+import java.util.List;
+
+import org.junit.Test;
+
+public class AppActivitiesDtoTest
+{
+	@Test
+	public void getActivitiesSorted_default_returnsActivitiesOrderedByStartTime()
+	{
+		ZonedDateTime now = ZonedDateTime.now();
+		ZonedDateTime t1 = now;
+		ZonedDateTime t2 = t1.plusSeconds(15);
+		ZonedDateTime t3 = t2.plusSeconds(15);
+		ZonedDateTime t4 = t3.plusSeconds(15);
+		AppActivitiesDto.Activity firstActivity = new AppActivitiesDto.Activity("Poker App", t1, t2);
+		AppActivitiesDto.Activity secondActivity = new AppActivitiesDto.Activity("Poker App", t3, t4);
+		AppActivitiesDto sut = new AppActivitiesDto(now, new AppActivitiesDto.Activity[] { secondActivity, firstActivity });
```

Comment:
What does ``sut`` mean?

Reply:
System under test. It's a naming convention we use in our team at work that slipped in, I'll rename it... 😄